.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'bixal-site-drupal - bug: '
```

Comment:
**question**: BSD a better fit here? It's a standard convention for commits and branch names.

Reply:
That's an interesting question. I definitely think it's good to add bug in the issue title and in the labels (we should totally add a feature request label too for your below template) for visibility. But putting the site name in the title of every issue might take up a lot of real estate when both the submitter and the reviewer of the issue know where they are already. I would say BSD or not needed.

Reply:
> @mattsqd thanks for the feedback. Added the `enhancement` label in [8b47d16](https://github.com/Bixal/bixal-site-drupal/pull/296/commits/8b47d16cb1d63baae2e6b63a3399c1a2d7336b5b).
> 
> The site label might be useful as we're maintaining multiple GitHub projects. You can quickly distinguish which repo it's coming from. Shall I remove that for this initial version of the PR?

I see, that makes sense. Feel free to leave as is and we can see how it works and go from there!